utils_cv/common/gpu.py

```diff
@@ -81,3 +77,11 @@ def system_info():
             print(f"{i}: {cuda.get_device_name(i)}")
     else:
         print("CPUs only, no GPUs found")
+
+
+def get_gpu_str():
+    if cuda.is_available():
+        devices = [str(x) for x in range(cuda.device_count())]
+        return ",".join(devices)
+    else:
+        return "-1"  # cpu
```

Comment:
Maybe for now, shall we move get_gpu_str() to the tracking code? Not sure how common/re-usable this is.